api/app/weather_models/fetch/predictions.py

```diff
@@ -138,10 +125,23 @@ async def fetch_latest_model_run_predictions_by_station_code_and_date_range(sess
         day_start = vancouver_tz.localize(datetime.datetime.combine(day, time.min))
         day_end = vancouver_tz.localize(datetime.datetime.combine(day, time.max))
 
-        daily_result = get_latest_station_prediction_mat_view(
-            session, active_station_codes, day_start, day_end)
-        for timestamp, model_abbrev, station_code, rh, temp, bias_adjusted_temp, bias_adjusted_rh, bias_adjusted_wind_speed, bias_adjusted_wdir, precip_24hours, bias_adjusted_precip_24h, wind_dir, wind_speed, update_date in daily_result:
-
+        daily_result = get_latest_station_prediction(session, active_station_codes, day_start, day_end)
+        for (
+            timestamp,
+            model_abbrev,
+            station_code,
+            rh,
+            temp,
+            bias_adjusted_temp,
+            bias_adjusted_rh,
+            bias_adjusted_wind_speed,
+            bias_adjusted_wdir,
+            precip_24hours,
+            bias_adjusted_precip_24h,
+            wind_dir,
+            wind_speed,
+            update_date,
```

Comment:
Doesn't look like `update_date` is being used. Would you mind removing it?

Reply:
Done in: https://github.com/bcgov/wps/pull/4118/commits/8b6dd8c7c493bfa8a2fccf7a7113eab96fa44d5b